temporian/implementation/numpy/data/plotter.py

```diff
@@ -17,42 +19,52 @@ class Options(NamedTuple):
     max_points: Optional[int]
     min_time: Optional[duration.Timestamp]
     max_time: Optional[duration.Timestamp]
+    max_num_plots: int
 
 
 def plot(
     event: Union[List[NumpyEvent], NumpyEvent],
-    index: Optional[Union[tuple, Any]] = (),
+    indexes: Optional[Union[tuple, List[tuple]]] = None,
     backend: str = DEFAULT_BACKEND,
     width_px: int = 1024,
     height_per_plot_px: int = 150,
     max_points: Optional[int] = None,
     min_time: Optional[duration.Timestamp] = None,
     max_time: Optional[duration.Timestamp] = None,
+    max_num_plots: int = 20,
 ):
     """Plots an event.
 
     Args:
         index: The index of the event to plot. Use 'event.index' for the
-            list of available indices. If index=None, select arbitrarily
-            (non deterministically) an index to plot.
+            list of available indices. If index=None, plots all the indexes.
         backend: Plotting library to use.
         width_px: Width of the figure in pixel.
         height_per_plot_px: Height of each sub-plot (one per feature) in pixel.
         max_points: Maximum number of points to plot.
         min_time: If set, only plot events after min_time.
         max_time: If set, only plot events before min_time.
+        max_num_plots: Maximum number of plots to display. If more plots are
+          available, only plot the first "max_num_plots" ones and print a
+          warning.
     """
 
     if isinstance(event, list):

```

Comment:
nit: rename `event` argument to `events`

Reply:
Done